code/game/gamemodes/miniantags/guardian/types/healer.dm

```diff
@@ -121,15 +121,17 @@
 		return
 	to_chat(src, "<span class='danger'>You begin to warp [A]</span>")
 	if(do_mob(src, A, 50))
-		if(!A.anchored)
-			if(beacon) //Check that the beacon still exists and is in a safe place. No instant kills.
-				if(safety_turf(beacon))
-					new /obj/effect/temp_visual/guardian/phase/out(get_turf(A))
-					do_teleport(A, beacon, 0)
-					new /obj/effect/temp_visual/guardian/phase(get_turf(A))
-				else
-					to_chat(src, "<span class='danger'>The beacon isn't in a safe location!</span>")
-			else
-				to_chat(src, "<span class='danger'>You need a beacon to warp things!</span>")
-	else
-		to_chat(src, "<span class='danger'>You need to hold still!</span>")
+		if(A.anchored)
+			to_chat(src, "<span class='danger'>You need to hold still!</span>")
```

Comment:
This one belonged to the `else` of the `if(do_mob(src, A, 50))` check. The anchor one had no message